README.md

```diff
@@ -62,6 +62,20 @@ type Config struct {
 }
 ```
 
+### Backend tag
+
+As we already said, Confita will call each backend one after another. But in order to avoid some useless processing we can specify in which backend the value of a field can be found with the `backend` tag:
```

Comment:
`As we already said` is a bit heavy, also we should avoid the `we`.

Maybe something like this ?:
```
By default, Confita queries each backend one after another until a key is found. However, in order to avoid some useless processing the `backend` tag can be specified to describe in which backend this key is expected to be found.
This is especially useful when the location of the key is known beforehand.
```